scripts/move_uploaded_crams.py

```diff
@@ -5,6 +5,11 @@
 import subprocess
 
 output = os.getenv('OUTPUT')
-assert output and output.startswith('gs://cpg-tob-wgs-archive/')
+assert output and output.startswith('gs://cpg-tob-wgs-archive/cram/batch')
```

Comment:
Should it be `batch` or `batch0`?

Reply:
Good spotting :). This doesn't contain the `0` so we can run it for future batches too :)

Reply:
Ah, gotcha. So in the command line, you would set it to `gs://cpg-tob-wgs-main/gvcf/batch0/`. Same for `move_uploaded_gvcfs.py`.

Reply:
That's exactly right.